src/Tooltip.js

```diff
@@ -1,23 +1,23 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
-import TetherContent from './TetherContent';
-import { getTetherAttachments, mapToCssModules, omit, tetherAttachements } from './utils';
+import Popper from './PopperContent';
```

Comment:
I know we switched from `Tether` to `Popper`.  Keeping it as `PopperContent` makes sense to avoid confusion with main popper.js library; however, since this is not really an exposed component persay, it's cleaner to just use it as `Popper` as you did here. If so ... should we rename `./PopperContent` to `Popper`. Otherwise, maybe keep the class name as `PopperContent` for consistency.